lib/styles/react-joyride.scss

```diff
@@ -196,7 +196,7 @@ $joyride-tooltip-arrow-scale: ($joyride-tooltip-arrow-size / ($joyride-tooltip-a
 
     &.bottom-right,
     &.top-right {
-      .triangle {
+      .joyride-tooltip__triangle {
```

Comment:
Selector `joyride-tooltip__triangle` should be written in lowercase with hyphens